examples/__testfixtures__/styled-components-to-ucl/unsupported.output.ts

```diff
@@ -1,9 +1,6 @@
 import { Box, FormControl, Input, Text } from '@rbilabs/universal-components';
 
-export const Scrollable = Box.withConfig({
-  // TODO: RN - unsupported CSS
-  // overflowX: 'auto',
-});
```

Comment:
do you think we should just default to commenting these out? I suspect that we may need to account for this overflow?

Reply:
Will change